### ISSUE TITLE
Add DV200 value to the Tapestation step in RNA RC

### ISSUE THREAD

**Issue by linnealafleur** [Clinical-Genomics/cg_lims#393]
Lims Issue
---------------

For definitions of headings below see [Basic Concepts](https://atlas.scilifelab.se/infrastructure/lims/basic_concepts/).

**Work Flow:** RNA
**Protocol:** Initial QC (RNA) v2
**Step:** Tapestation Reception Control (RNA)

**Description:** 
We want to add an additional QC value called DV200 to the RNA samples from Tapestation. 
A new UDF called "DV200" and an additional placeholder for a csv file called Compact Region Table CSV is needed.
To the DV200 udf the value from the column "% of Total" should be fetched for each sample.

Example file:
[01-RNA_compactRegionTable.csv](https://github.com/Clinical-Genomics/cg_lims/files/11914837/01-RNA_compactRegionTable.csv)

**We need:**
- [x] An EPP "Get DV200" to get the "% of Total" values or update the EPP "Get RIN from file" @Karl-Svard 
- [x] A sample udf "DV200"
- [x] A placeholder "Compact Region Table CSV"
- [x] Create new process udfs "Copy task 5 - Source Field" with "DV200" and "Copy task 5 - Source Step" with "Tapestation Reception Control (RNA)", and sample udf "DV200" in master step _Aggregate QC (RNA) v1_

Extra: it would be nice to have a checkbox UDF in the Tapestation step called "DNA contamination" so we can record which samples seems to be contaminated. - Added 240208/IL

**Comment by amandaraud** [comment]
This issue is related to https://github.com/Clinical-Genomics/Lab-Production/issues/560

**Comment by linnealafleur** [comment]
@Karl-Svard @idalindegaard @amandaraud what is the status for this?
Also we do not want to remove RIN. At first we want to collect both to be able to compare them.

**Comment by Karl-Svard** [comment]
It was on hold a bit as recent lims development time has been limited. But I heard that work to compare these two values is planned to start quite soon? If that's the case then we can bump the prio of this up

**Comment by linnealafleur** [comment]
> It was on hold a bit as recent lims development time has been limited. But I heard that work to compare these two values is planned to start quite soon? If that's the case then we can bump the prio of this up

Well yes it would be good to be able to start collecting data, since we need to gather some before we can do the comparison.

**Comment by idalindegaard** [comment]
Changes done on stage
- A new EPP called "Get DV200 From File", new udfs "DV200" on Derived sample and Measurement level, and a placeholder "Compact Region Table (DV200)" in Tapestation Reception Control (RNA)
- "Copy task 5 - Source Field" with "DV200" and "Copy task 5 - Source Step" with "Tapestation Reception Control (RNA) v1", and sample udf "DV200" in master step Aggregate QC (RNA) v1

<img width="735" alt="Screenshot 2024-02-09 at 15 14 42" src="https://github.com/Clinical-Genomics/cg_lims/assets/82438141/0bac82fc-8739-41d6-933d-594d2cc1528b">
<img width="735" alt="Screenshot 2024-02-09 at 15 17 16" src="https://github.com/Clinical-Genomics/cg_lims/assets/82438141/a1546137-3c18-4904-a414-0ab0e275a491">

This worked, the values are fetched from the CSV file and the values are recorded in the Aggregation step

**Comment by idalindegaard** [comment]
Updated LIMS prod and merged the RC document PR with the changes
